src/Options/Definitions.js

```diff
@@ -353,6 +353,22 @@ module.exports.ParseServerOptions = {
     env: 'PARSE_SERVER_REST_API_KEY',
     help: 'Key for REST calls',
   },
+  oauth: {
+    env: 'PARSE_SERVER_OAUTH',
+    help: 'Sets whether to use the OAuth protocol',
+    action: parsers.booleanParser,
+    default: false,
+  },
+  oauthKey: {
+    env: 'PARSE_SERVER_OAUTH_KEY',
+    help: 'Key for OAuth protocol',
+  },
+  oauthTTL: {
+    env: 'PARSE_SERVER_REST_API_KEY',
```

Comment:
This looks like the wrong env variable. Please follow the guide to generate these definitions

https://github.com/parse-community/parse-server/blob/master/CONTRIBUTING.md#parse-server-configuration